tests/tt_metal/tt_metal/unit_tests/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ set(UNIT_TESTS_SRC
     ${CMAKE_CURRENT_SOURCE_DIR}/multichip/device_cluster_api.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/multichip/erisc_app_direct_send.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/multichip/ring_gather_kernels.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tt_stl/any_range.cpp
```

Comment:
What do you think if add _test name to the test cpp files.
Right now you added:
any_range.cpp and any_range.hpp but any_range.cpp is the test.

Reply:
Fixed